BT-Tracking/Interface/Current Data/CurrentDataVC.swift

```diff
@@ -34,9 +34,11 @@ final class CurrentDataVC: UIViewController {
 
         viewModel.obervableErrors.subscribe(onNext: { [weak self] error in
             self?.hideProgress()
+            #if DEBUG
```

Comment:
Nepridavej sem ten DEBUG. Je to schvalne pro pridat, ze uzivatel jeste nema ani jedno stazeny aktualnich dat a request mu skonci v chybe. Tak aby tam nebyla jen prazdna obrazovka a pochopil ze nastala chyba. Pri dalsim otevreni obrazovky se znovu pokusi o stazeni dat

Reply:
Tak by tam chtelo dat nakej "lidstejsi" text

Reply:
Tak tam muzeme zobrazit `unknown` error.